ross/plotly_theme.py

```diff
@@ -0,0 +1,377 @@
+            "aaxis": {"gridcolor": "#DFE8F3", "linecolor": "#A2B1C6", "ticks": ""},
+            "baxis": {"gridcolor": "#DFE8F3", "linecolor": "#A2B1C6", "ticks": ""},
+            "bgcolor": "white",
+            "caxis": {"gridcolor": "#DFE8F3", "linecolor": "#A2B1C6", "ticks": ""},
+        },
+        "title": {"x": 0.05},
+        "xaxis": {
+            "automargin": True,
+            "gridcolor": "#EBF0F8",
+            "showline": True,
+            "linecolor": "black",
+            "linewidth": 2.0,
+            "mirror": True,
+            "ticks": "",
+            "title": {"standoff": 15},
+            "zeroline": False,
+            "zerolinecolor": "#EBF0F8",
+            "zerolinewidth": 2,
+        },
+        "yaxis": {
+            "automargin": True,
+            "gridcolor": "#EBF0F8",
+            "showline": True,
+            "linecolor": "black",
+            "linewidth": 2.0,
+            "mirror": True,
+            "ticks": "",
+            "title": {"standoff": 15},
+            "zeroline": False,
+            "zerolinecolor": "#EBF0F8",
+            "zerolinewidth": 2,
+        },
+    },
+    data={
+        "bar": [
+            {
+                "error_x": {"color": "#2a3f5f"},
+                "error_y": {"color": "#2a3f5f"},
+                "marker": {"line": {"color": "white", "width": 0.5}},
+                "type": "bar",
+            }
+        ],
+        "barpolar": [
+            {"marker": {"line": {"color": "white", "width": 0.5}}, "type": "barpolar"}
+        ],
+        "carpet": [
+            {
+                "aaxis": {
+                    "endlinecolor": "#2a3f5f",
+                    "gridcolor": "#C8D4E3",
+                    "linecolor": "#C8D4E3",
+                    "minorgridcolor": "#C8D4E3",
+                    "startlinecolor": "#2a3f5f",
+                },
+                "baxis": {
+                    "endlinecolor": "#2a3f5f",
+                    "gridcolor": "#C8D4E3",
+                    "linecolor": "#C8D4E3",
+                    "minorgridcolor": "#C8D4E3",
+                    "startlinecolor": "#2a3f5f",
+                },
+                "type": "carpet",
+            }
+        ],
+        "choropleth": [
+            {"colorbar": {"outlinewidth": 0, "ticks": ""}, "type": "choropleth"}
+        ],
+        "contour": [
+            {
+                "colorbar": {"outlinewidth": 0, "ticks": ""},
+                "colorscale": [
+                    [0.0, "#0d0887"],
+                    [0.1111111111111111, "#46039f"],
+                    [0.2222222222222222, "#7201a8"],
+                    [0.3333333333333333, "#9c179e"],
+                    [0.4444444444444444, "#bd3786"],
+                    [0.5555555555555556, "#d8576b"],
+                    [0.6666666666666666, "#ed7953"],
+                    [0.7777777777777778, "#fb9f3a"],
+                    [0.8888888888888888, "#fdca26"],
+                    [1.0, "#f0f921"],
+                ],
```

Comment:
Here, it's the same idea. This color scale is blue to red (I suppose by testing the values), and we could replace the list by:
`"rdbu"` or `colors.sequential.RdBu`

Reply:
This one is just a copy from the plotly theme. To me it is ok if we replace it.
I will make a document with all plots that we have (ucs, campbell, frequency response etc.) so that we can see how the modifications to the plotly theme affects all figures and then I will test the modification and show it here.

Reply:
Got it, ok then